logrotate.8.in

```diff
@@ -524,8 +525,8 @@ Specifies which command to use to uncompress log files.  The default is
 .TP
 \fBcompressext\fR
 Specifies which extension to use on compressed logfiles, if compression
-is enabled.  The default follows that of the configured compression
-command.
+is enabled.  The default extension is .gz and compressext should be
+updated with an appropriate extension, if \fBcompresscmd\fR has been set.
```

Comment:
The default extension can be configured at build time via `configure --with-compress-extension=...`.  So this needs to be handled in the same way as `@DEFAULT_MAIL_COMMAND@`.

Reply:
Is *that* what is meant by "configured"...!  That's even worse, as the other three directives all call out gzip as the default in the man page.

Is the man page adjusted with the config options?

Reply:
I understand your solution now.

Reply:
We seem to have `AC_SUBST` for this already: https://github.com/logrotate/logrotate/blob/0c423148c613ec7fc52ac5786e06aead95d25785/configure.ac#L165

So we can just replace `.gz` with `@COMPRESS_EXT@` in `logrotate.8.in` and the installed man page will contain the correct information.

Reply:
But the other three directives will not match if it is configured differently, correct?